prac_06/guitar.py

```diff
@@ -0,0 +1,31 @@
+"""CP1404 Practical - Guitars!
+Estimated time: 40 minutes
+Actual time: 50 minutes """
+
+YEAR = 2023
+VINTAGE_THRESHOLD = 50
+
+
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        """
+        Initialise a Programming language instance.
+        name: string, name of the guitar
+        year: integer, year the guitar was first produced
+        cost: float, price of the guitar
+        """
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name} ({self.year}) : ${self.cost}"
```

Comment:
To format the self.cost output as dollars and cents, we can use `{self.cost:,.2f}`

Reply:
Thanks, fixed.